### PR TITLE
API Billing Fixes

### PR DESCRIPTION
**Pull request by albertcui**
- disable keys when payment fails
- allow people to create new keys if previous payment fails ONLY if no outstanding balance

### CODE REVIEW

**Comment by howardchung** [comment]
Can you split the formatting changes into their own PR?

File: routes/README_FOR_API_KEYS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Make sure to set these ENV variables:
+
+STEAM_API_KEY=761FE7D69D5546AB1C053A911F14AF08
```
Comment (howardchung) [inline routes/README_FOR_API_KEYS.md:3]: Is this a real key?

File: svc/apiadmin.js [hunk 2]
```diff
@@ -87,92 +87,91 @@ function storeUsageCounts(cursor, cb) {
...
+          `
+                  UPDATE api_keys SET is_canceled = true WHERE subscription_id = ?
+                `,
+          [sub.id]
+        );
 
-  if (cursor) {
-    options.starting_after = cursor;
-  }
+        continue;
```
Comment (howardchung) [inline svc/apiadmin.js:114]: Are we letting the background job remove the canceled key from redis?

**Review by howardchung — APPROVED** [review]

**Comment by howardchung** [comment]
If the user has an outstanding balance, do they have a way of paying it from our UI?

File: svc/apiadmin.js (hunk 2 above)
Comment (albertcui) [reply svc/apiadmin.js:114]: updateAPIKeysInRedis deletes all keys so just let that take care of it

**Pull request merged**